test-apps/display-performance-test-app/.gitignore

```diff
@@ -0,0 +1,3 @@
+.vite-inspect/
```

Comment:
dont necessarily need to create a new .gitignore for this
couldnt you add this to the one at the root?

Reply:
done, I also updated the .gitignore in DTA